### PR TITLE
BoxStation Armory update [BETTER REINFORCED] [GONE SEXUAL?] 

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Redoes armory to be better secure, the main point being that it takes longer to steal individual items so security has time to respond . There's also a space airlock right in equipment room to easily be able to get into space, the hallway option made it a bit _too_ easy to respond to everything, this a compromise.

chrome_CxUfOQnQaK

chrome_CgZR1JaOO0

# Wiki Documentation

BoxStation Armory and Equipment Room look different.

# Changelog

:cl:  ToasterBiome, JamieD12, Lyis, MajesticManatee
mapping: BoxStation Armory/Equipment Room has been extended with better protection, along with a new space airlock.
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
JamieSeal

**Review by JamieD1 — APPROVED** [review]

**Comment by iamthesilly** [comment]
do all those windoors require armory access? because if so then to open the armory in case it's needed would require opening every single windoor as well instead of just hitting the button for the shutters, this takes valuable time which you usually don't want to waste in situations where the armory needs to be opened

**Comment by Cartlord** [comment]
Could make the "open armory" button also open every windoor
I think making every windoor be a reinforced windoor might be pushing how secure this needs to be - why not standard? It'd still be a pain to break ALL of them.

**Comment by Hopekz** [comment]
No-one really mentioned this so idk if I'm stating the obvious but this is now the easiest way in to the armory with these changes so this change should be considered with this entry in mind.

Even gets you the furthest from beepsky

If you hug the wall to get to this location you also don't trip any motion alarms outside too.

Also spaces the locker rooms so people can't get to the space suits..

image

**Comment by patpol4** [comment]
You could rearrange to move the lathe row so it doesnt block a clear stun/laser shot inside

**Comment by Bibby0110** [comment]
Not a big fan if the windoors are access locked

File: _maps/map_files/YogStation/YogStation.dmm [hunk 1]
```diff
@@ -25145,6 +24819,28 @@
...
+	pixel_y = 3
+	},
+/obj/item/gun/ballistic/shotgun/riot,
+/obj/structure/window/reinforced{
+	dir = 8;
+	max_integrity = 350
+	},
+/obj/machinery/door/window/brigdoor/northright{
+	name = "Armory Door";
+	req_access_txt = "3"
```
Comment (ynot01) [inline _maps/map_files/YogStation/YogStation.dmm]: The windoors appear to all be armory locked
`#define ACCESS_ARMORY 3`

Comment (JamieD1) [reply _maps/map_files/YogStation/YogStation.dmm]: Yep we can fix that, we are also adding window shutters as well and removing the window on the right.

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Awaiting a couple additions

**Comment by patpol4** [comment]
will review after the additions

**Comment by DukeBobRoss** [comment]
image

You should reinforce these doors as well as the maint door. The armory gets stolen by people running in with an RCD or Emag more than from space imo.

File: code/game/machinery/camera/motion.dm [hunk 2]
```diff
@@ -76,6 +79,9 @@
 		if(isliving(AM))
 			newTarget(AM)
 
+/obj/machinery/camera/motion/armoury
```
Comment (patpol4) [inline code/game/machinery/camera/motion.dm]: armory

File: code/game/machinery/camera/motion.dm [hunk 3]
```diff
@@ -66,7 +66,10 @@
 	for (var/mob/living/silicon/aiPlayer in GLOB.player_list)
 		if (status)
 			aiPlayer.triggerAlarm("Motion", get_area(src), list(src), src)
-			visible_message(span_warning("A red light flashes on the [src]!"))
+	visible_message(span_warning("A red light flashes on the [src]!"))
+	if(armoury)
```
Comment (patpol4) [inline code/game/machinery/camera/motion.dm]: armory

File: code/game/machinery/camera/camera.dm [hunk 4]
```diff
@@ -84,6 +86,13 @@
 		LAZYADD(myarea.cameras, src)
 	proximity_monitor = new(src, 1)
 
+	if(armoury)
+		radio = new()
+		radio.set_frequency(FREQ_SECURITY)
+		radio.use_command = TRUE
+		radio.independent = TRUE
+		radio.name = "Armoury"
```
Comment (patpol4) [inline code/game/machinery/camera/camera.dm]: armory

File: code/game/machinery/camera/camera.dm [hunk 5]
```diff
@@ -44,6 +45,7 @@
 
 	//Reference to the obj/mob we're built into
 	var/built_in
+	var/armoury = FALSE
```
Comment (patpol4) [inline code/game/machinery/camera/camera.dm]: armory

File: code/modules/paperwork/paper_premade.dm [hunk 6]
```diff
@@ -43,6 +43,11 @@
 	info = "<b><u>Setting up your paystand</u><br>Step 1</b><br>Locate your department budget card.<br><b>Step 2</b><br>Scan your department budget card onto the paystand."
 	infolang = /datum/language/common
 
+/obj/item/paper/guides/jobs/security/donut
+	name = "paper- 'Weapon Crate'"
+	info = "<B>Regarding the Weapons Crate:</B><BR>\nIf you are reading this, you have found me out, I was on shift once and on my watch someone came and stole the guns from this crate.. I have attempted to get away with this, so please keep this quiet. Here is a donut for your trouble! Thanks"
```
Comment (patpol4) [inline code/modules/paperwork/paper_premade.dm]: Thanks.

File: code/game/machinery/camera/camera.dm (hunk 5 above)
Comment (JamieD1) [reply code/game/machinery/camera/camera.dm]: Armoury

File: code/game/machinery/camera/motion.dm (hunk 2 above)
Comment (JamieD1) [reply code/game/machinery/camera/motion.dm]: Armoury

File: code/game/machinery/camera/motion.dm (hunk 3 above)
Comment (JamieD1) [reply code/game/machinery/camera/motion.dm]: armoury

File: code/game/machinery/camera/camera.dm (hunk 5 above)
Comment (JamieD1) [reply code/game/machinery/camera/camera.dm]: armoury

File: code/modules/paperwork/paper_premade.dm (hunk 6 above)
Comment (JamieD1) [reply code/modules/paperwork/paper_premade.dm]: Thanks

**Comment by UltraBigRat** [comment]
Windoor spam on this level is pretty horrid.

**Comment by ReddicusDragon** [comment]
remove orcas sprites

**Comment by KillerOrcaCora** [comment]
image
You guys really suck at this

**Pull request merged**